### DISCUSSION TITLE
GitHub Azure DevOps Speed Test

### DISCUSSION THREAD

**Discussion by JanuszNowak** [AzureCommunityPL/group#478] [Inspiracje]
#inspiracje Rozwiązania pozwalającego odpowiedzieć na pytanie, który region Azure DevOps/GitHub ma najmniejsze opóźnienia sieciowe. Jest to fork istniejącego rozwiązania, tylko zamiast storage accountów odpytywana są repozytoria git.
https://azure4devops.com/GithubAzureDevOpsSpeedTest/

![image](https://user-images.githubusercontent.com/5168275/224538513-1ffa7e2d-1432-40dc-8226-743d9831110b.png)

#AzureDevOps #Github
#GithubAzureDevOpsSpeedTest

**Comment by sbugalski** [comment]
Czyli z Githuba odpytywany jest Github?

**Reply by JanuszNowak** [reply]
@sbugalski wszystkie end pointy są odpytywanie z maszyny, na której otworzysz stronę, czyli z twojego komputera z twojej lokalizacji.

**Reply by sbugalski** [reply]
Mucias gracias, nie mogłem się tego doszukać, a na czytanie kodu było już za późno.
Dzięki za wkład! :)
